ram_msgs/AdditiveManufacturingPose[] poses
---
ram_msgs/AdditiveManufacturingPose[] poses
